my_game.py

```diff
@@ -254,9 +254,16 @@ def on_update(self, delta_time):
         elif self.right_pressed and not self.left_pressed:
             self.player_sprite.angle -= PLAYER_ROTATE_SPEED
 
+        # Player rocket engine
+        if self.up_pressed:
+            # rotate player graphics to match direction
+            self.player_sprite.change_x += self.player_sprite.speed * cos(self.player_sprite.radians + pi/2)
+            self.player_sprite.change_y += self.player_sprite.speed * sin(self.player_sprite.radians + pi/2)
+        
```

Comment:
We should probably have a global variable _PLAYER_THRUST_ which should replace _self.speed_ used here.

Reply:
is fixed?

Reply:
Is berry fixed